script/fs-bench/work/reset.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set -euo pipefail
+
+CONTAINERD_ROOT=/var/lib/containerd/
+CONTAINERD_CONFIG_DIR=/etc/containerd/
+REMOTE_SNAPSHOTTER_SOCKET=/run/containerd-stargz-grpc/containerd-stargz-grpc.sock
+REMOTE_SNAPSHOTTER_ROOT=/var/lib/containerd-stargz-grpc/
+REMOTE_SNAPSHOTTER_CONFIG_DIR=/etc/containerd-stargz-grpc/
+
+NOSNAPSHOTTER="${1:-}"
+NOCLEANUP="${2:-}"
+SCRAPE_OUTPUT_FILENAME="${3:-}"
```

Comment:
We should receive these switches through environment variables or allow random order of `-`-prefixed options. Environment variables are easier maybe.